fe/src/main/java/org/apache/doris/alter/SchemaChangeJobV2.java

```diff
@@ -184,13 +184,22 @@ protected void runPendingJob() throws AlterCancelException {
                 totalReplicaNum += tablet.getReplicas().size();
             }
         }
-        MarkedCountDownLatch<Long, Long> countDownLatch = new MarkedCountDownLatch<Long, Long>(totalReplicaNum);
+        MarkedCountDownLatch<Long, Long> countDownLatch = new MarkedCountDownLatch<>(totalReplicaNum);
         db.readLock();
         try {
             OlapTable tbl = (OlapTable) db.getTable(tableId);
             if (tbl == null) {
                 throw new AlterCancelException("Table " + tableId + " does not exist");
-            } 
+            }
+
+            boolean isStable = tbl.isStable(Catalog.getCurrentSystemInfo(),
+                    Catalog.getCurrentCatalog().getTabletScheduler(),
+                    db.getClusterName());
+            if (!isStable) {
+                LOG.warn("doing schema change job while table is not stable, wait until tablet to be repaired.");
```

Comment:
You can add this error msg to the `errMsg` of the AlterJob, so that user can see what happen.
And this log should contains job id for debugging. And in `errMsg`, it can be shorten, eg, "table is unstable".